android/core/src/main/java/com/pega/vehicledamagemodeling/VehicleDamageModeling.java

```diff
@@ -39,50 +41,65 @@
 import static com.badlogic.gdx.graphics.g3d.attributes.ColorAttribute.AmbientLight;
 
 public class VehicleDamageModeling extends ApplicationAdapter {
-    private PerspectiveCamera perspectiveCamera;
-    private LimitedCameraInputController cameraController;
     private ModelBatch modelBatch;
+    private Environment environment;
+    private PerspectiveCamera camera;
+    private LimitedCameraInputController cameraController;
+    private SelectionService selectionService = new SelectionService(new SelectedPartsRepository(), new Parser());
     private AssetManager assets;
     private Array<ModelInstance> instances = new Array<>();
-    private Environment environment;
-    private boolean loading;
-    private SelectionService selections = new SelectionService(new SelectedPartsRepository(), new Parser());
     private final VehicleDamageReportCallback callback;
+    private JsonObject jsonWithSelectedParts;
+
+    private boolean loading;
+
```

Comment:
Te puste linie miedzy polami sa do usuniecia. Mozesz jeszcze przerzucic callback nizej bo to final